src/analysis/CMakeLists.txt

```diff
@@ -530,6 +530,13 @@ if (WITH_PDAL AND PDAL_2_5_OR_HIGHER)
   endif()
 endif()
 
+if (WITH_SFCGAL)
```

Comment:
The correct way to handle this is to always expose the algorithm, but at runtime raise a processing exception of the required library is not available (with an explanatory message).


This let's users still work on models using these tools, and gives them a friendly explanation as to why the to won't work (as opposed to it just getting missing)

Reply:
I agree with that. However, I reproduced the existing logic of the PDAL processings. I QGIS is built without PDAL support the processing are not exposed.

Reply:
https://github.com/qgis/QGIS/issues/63106

Reply:
Done.